package/MDAnalysis/lib/distances.py

```diff
@@ -96,8 +149,15 @@
     _distances['openmp'] = importlib.import_module(".c_distances_openmp",
                                           package="MDAnalysis.lib")
 except ImportError:
-    pass
-del importlib
+    del importlib
+
+if HAS_DISTOPIA:
+    import importlib
```

Comment:
I think I'm missing something obvious, why is importlib being moved inside of the try/except here? So it only now dels it _if_ there's an import error?

Wouldn't it make more sense to do like we did before, del at the end after the attempt to import everything?